skedulr/app/controllers/shifts_controller.rb

```diff
@@ -2,20 +2,34 @@ class ShiftsController < ApplicationController
   include ShiftsHelper
 
   def index
-    @first_name = current_employee.first_name
-    @employee = current_employee
+    if session[:manager]
```

Comment:
I'm likely missing something here, but why can't we just use current_employee regardless of the :manager session value?